### ISSUE TITLE
Feature Request: disable logging fo drivers.

### ISSUE THREAD

**Issue by adavt** [ioBroker/ioBroker#328]
The minimum possible level of logs is "error". Sometimes in case of not quite correct operation of the equipment, it is useful to be able to completely disable logs on any driver. To avoid seeing persistent errors. About which it is already known.

**Comment by Apollon77** [comment]
Hm ... Why we should prevent errors from ebing shown? If adpters have problems  then these need to be reported and fixed. Do you have examples?

**Comment by adavt** [comment]
> Hm ... Why we should prevent errors from ebing shown? If adpters have problems then these need to be reported and fixed. Do you have examples?

I have a modbus instance that polls the wrong device. Nothing can be fixed there - the features of the device itself. The same problems were with this device and when connecting to a non- IoB. Sometimes, about once a minute, IoB will send in log an incorrect answer "(8355) Client in error state.". Then it works normally for next 1-2 minutes. I know about this and I understand that nothing can be changed. But huge lists of errors that are uninteresting prevent you from taking the necessary warnings. It is best to turn off what is unnecessary.

**Comment by aski71** [comment]
Let's say your adapter runs perfectly and serves your needs. There's just a datapoint that isn't really used yet which is causing an error being written every 2 seconds. You reach out to the developer for a fix. It takes time, but you can't use the adapter at all in the meantime because it's cluttering your disk. 
So yes: If an error is identified and sent to the developer, I'd like to disable the error log for this adapter if it isn't affecting my use case and don't want to be forced to wait until a fix comes in. Which may or may not be in months time, depending on the availability of the developer.
So why render a whole adapter useless for the user by not allowing him to disable logging entirely? 
I have the exact same situation right now.

**Comment by aski71** [comment]
In addition: Not being able to disable the known and for the time being accepted error log also prevents me from immediately seeing other, maybe more important logs.

**Comment by Apollon77** [comment]
So in fact you want a "loglevel above error" like "disabled" ?

**Comment by fortschritt311** [comment]
Hi.
I would also need this feature.
I use the Modbus adapter for my inverter. If no power is produced at night, the inverter switches off and with it the flashed ShineLanX stick. The adapter now writes to the log all night long:

```
modbus.0 2024-06-12 22:23:57.637 error Client in error state.
modbus.0 2024-06-12 22:23:57.636 error Socket Error
```

Regards Maik

**Comment by Apollon77** [comment]
@fortschritt311 or request at the adater that the loglevel gets changed and you can increase to error?

**Comment by fortschritt311** [comment]
@Apollon77 thanks, I have placed a feature request with modbus adapter. But I think it could also be interesting for other adapters so that the function is centralized.

https://github.com/ioBroker/ioBroker.modbus/issues/525

**Comment by aski71** [comment]
Yes. A centralized "disabled" above the "error" level would be handy.
